docs/core/integration-quality-scale/rules/iqs035.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "IQS035 - Service actions are registered in async_setup"
+---
+
+IQS035 is as follows:
+
+Service actions are registered in async_setup
+
+## Reasoning
+
+Integrations can add their own service actions to Home Assistant.
+In the past, they have been frequently registered in the `async_setup_entry` method and removed in the `async_unload_entry` method.
+The result of this is that the service actions are only available when there is a loaded entry.
+This is not ideal, since this way we can't validate automations users create that use these service actions, since it is possible that the config entry could not be loaded.
```

Comment:
Should we call it configuration entry everywhere? I don't remember how consistent we've been in the other rules.

Reply:
I'll write down that I will do a search afterwards